src/components/Forms/SignupForm/index.js

```diff
@@ -1 +1 @@
-export { default } from "./SignupForm";
+export { default } from "./SignupFormB";
```

Comment:
@Jennimar17 This is not correct since we need to keep showing the old signup page. I'm fixing in a new commit.